src/testsuite/vec_perf_i128.c

```diff
@@ -124,4 +157,81 @@ int timed_mulluq (void)
   return rc;
 }
 
+#pragma GCC optimize ("no-unroll-loops")
+//#define __DEBUG_PRINT__
+int
+timed_muludq (void)
+{
+  vui128_t i, j, k;
+  int ii, iii;
```

Comment:
really?  :-)  not a big fan of these variable names, but will look the other way.  ;-)
these appear elsewhere as well